floodviz/static/js/map.js

```diff
@@ -1,91 +1,42 @@
-document.addEventListener("DOMContentLoaded", function (event) {
-	"use strict";
+"use strict";
 
-	// Define helper functions
-	function degreesToRadians(degrees) {
-		return degrees * Math.PI / 180;
-	}
-
-	function radiansToDegrees(radians) {
-		return radians * 180 / Math.PI;
-	}
-
-	// read in projection information
-	var proj = proj4(FV.mapinfo.proj4string);
+// define module for map interactios
+FV.mapmodule = function (options) {
 
+	var self = {};
+	var height = options.height;
+	var width = options.width;
+	var proj = options.proj;
+	var sel_div = options.div_id;
 	var project = function (lambda, phi) {
 		return proj.forward([lambda, phi].map(radiansToDegrees));
 	};
-
 	project.invert = function (x, y) {
 		return proj.inverse([x, y]).map(degreesToRadians);
 	};
-
 	//Define map projection
 	var projection = d3.geoProjection(project);
-
 	// Give projection initial rotation and scale
-	projection
-		.scale(1)
-		.translate([0, 0]);
-
+	projection.scale(1)	.translate([0, 0]);
```

Comment:
There's a tab character between `projection.scale(1)` and `.translate([0, 0]);`
There should either be no space or a line break after `scale` with `.translate` indented.